### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by ozh**

### CODE REVIEW

**Comment by ozh** [comment]
We will **never** have an automated `awesome-lint` nazi script enforcing dumb rules :grin: . Just leaving this PR open so it's easy to re-run the action if we ever feel the need for it, though.

Idea to consider: maybe have a cron based action to check every link once in a while ?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-  uses: jthegedus/github-action-awesome-lint@v0.1.0
+workflow "Lint Awesome List" {
+  on = "push"
+  resolves = ["max/awesome-lint"]
+}
+
+action "max/awesome-lint" {
+  uses = "max/awesome-lint@1.0.0"
+}
+
```
Comment (LeoColomb) [inline .github/workflows/main.yml]: 😱😉

**Comment by LeoColomb** [comment]
@ozh I've just added cron trigger to run the action on a scheduled base.
Also don't forget to add dependabot to keep actions up to date. 👍 

---
I'm not sure it's a really good idea to add a lint action even on a cron base.
It'll fail almost each time, might make maintenance very time consuming.
I've seen quite a lot of awesome-* repo completely ignore this kind of tests because make things too complicated... 

🤷‍♂️

**Comment by ozh** [comment]
@LeoColomb I meant something cron based  to check links for 404s, not cron for awesome-lint :)

**Comment by LeoColomb** [comment]
@LeoColomb Not sure to follow, awesome-lint check for dead links, so with this we have a cron based to check links, right?

**Comment by ozh** [comment]
Oh, never mind then. I thought awesome-lint was just checking the syntax and reporting when the readme doesn't follow some guideline. Perfect then :)

**Comment by LeoColomb** [comment]
@ozh 👍 Merge then?

**Comment by ozh** [comment]
Alternative, will check later: https://github.com/dkhamsing/awesome_bot (example of use: https://github.com/ziadoz/awesome-php, with link whitelist)

**Pull request merged**